### ISSUE TITLE
[Long list] Bugs / Small suggestions

### ISSUE THREAD

**Issue by Kimod** [Mordonus/MALClient#42]
First of all, I really love your app, like, really ^^
So after using it for a while I noticed a few things that bothered me, so I figured I would write them here, so they could get solved.

**Bugs**
* Shows show average MAL rating only after I clicked on them - http://puu.sh/q7pid.png
 *  ✔️ Scores are cached and will populated if the value is known. Api limitation of sorts.
* When there is back button on top, and I make search bar available, like if I'm on someone's profile and search something, the search bar covers the back button http://puu.sh/q7pAm.png
 * ✔️ Fixed
* Top anime can't load more than 250 shows
 * ✅  Works as intended, explanation below
* Gender, Time I Joined and Location in profile shows information from different things - http://puu.sh/q7rgB.jpg
 * ✔️ Fixed
* When I click on the author of the Article, it opens his profile in browser
 * ✔️ Fixed
* When I browse the "Top Anime" page and click at show I don't have in my list and look at the amount of episodes it shows a "?"
 * ✔️ Fixed
* Show title is in rare cases overlaping the number of the show on the list - http://puu.sh/q7uVT.jpg
 * ✔️ Fixed, gave some love to those detailed grid items.
* If I click "Anime Search" more than one time and then go to the "Anime list", search bar remains open
 * ✔️ Fixed
* When I have selected one show and press "Select one at random", it does show random show on the right but it doesn't select it in the list
 * ✔️ Fixed, I did try to make it allow to use arrows to naviagte accross items but there's something with stock control and it will accept arrow movements after a few presses.
* Anime Search is different than that on the website, it shows the shows differently, http://puu.sh/q7wPo.jpg the app didn't even list on the list Space Dandy, which on MAL is first
 * ✅ This is what I get from MAL's API response nothing I can do here.
* In Settings, when you click on drop down list, it makes the whole list move to the right http://puu.sh/q7xxi.png
 * ❓ Umm, you mean that it hides originating combo box?
	
**Mobile only bugs**
* There isn't "All" category in "Anime list", when I click to the right of the hamburger menu
 * ✔️ Fixed
* When I click on magnifier in "Anime list" i can't immediately type, while on deskop I can
 * ✔️ Fixed
* When you are viewing show details, then in the "General" tab when you grab and drag it diagonaly on shows details, like the amount of episodes or type of the show, you can drag it, I recomend looking at the picture http://puu.sh/q7sgO.jpg , it's hard to describe it, this also happens in the "Details" with Genres and Episodes, also in Recommendations details
 * ✔️ Fixed
	
**Suggestions**
* In reviews when I right click the review (or long hold it, or click on name, whatever), I could view profile of the writer
* When I right click/long hold on Image in Article in "Articles", I would like an option to save it
 * Pretty hard and there are more urging matters, maybe someday... ^^
* Add a way to load more reviews
 * ✔️ Upped the default number of pulled ones.
* Add the show popularity and ranked number to anime details
 *  💭  Data is scarce here but I'll try...
* Add the episode length, so I would know how long the film is or how short the shorts are
 * 💭  Same as above
* A way to copy shows title, like I would like to make the title of the show selectable http://puu.sh/q7tOl.jpg
 * ✔️ Done.
* A way to don't show "Manga list", "Manga search" and "Top Manga" in hamburger menu (maybe pointless, but I would use it ^^)
 * ✔️ Added.
* Add the show "airing day in the week" / "number of the show on list" to the list view, it is already in detailed grid view
 * ✔️ Added.
* Add all seasons to the "Seasonal Anime" section, all are on the website
* In list view on desktop when I have selected one show and press the down / up arrow, it moves but to make sure the viewer on right gets updated I have to press Enter, I would like to do it without pressing Enter
* Add a way to view someones profile (now we can only view his list and even now I don't like that "List Source" button, I think it would be better in hamburger menu, but thats just my opinion ^^)
 * Moved this button deeper (in the "more" section of bottom app bar)
* Add producer of the anime to the details of the anime
 * 💭 Again... missing data. I'll probably take this from html(duh) and redo details page in details page ^^

I hope that at least some of those bug fixes / suggestions will get implemented, so I could enjoy your app even more ^^ and also I hope, that some of those pictures helped to describe, what I meant

**Comment by Mordonus** [comment]
Wow, good job listing those! Some of these are just api limitations, there is no specific data on certain page etc... It's hard to get the data :) I'll elaborate a bit more when I get to my computer. Thanks a lot!

**Comment by Mordonus** [comment]
*Bugs*
1. There's no score in api response, I've to get details from search. I did cache this thing but something seems to have bugged out.
2. Got it!
3. This is somehow intentional, when scrolling deep into top entries you can find +18 stuff and I had once nertification failed because of that... (no age rating received from MAL)
4. Okay.
5. This is something I left for later.
6. Odd, I'll look into it.
7. I'll have to tweak this view a bit.
8. Okay
9. Didn't even think of it to be honest.
10. This is what MAL api returns I believe that those searches are different.
11. I don't know what going on here.

*Mobile bugs*
1. I probably forgot about this ^^
2. Got it
3. I underestand, I'll try to do something about it.

To be continued,,,

**Comment by Mordonus** [comment]
Bugs have been fixed, I'll look at suggestions tomorrow :)

**Comment by Kimod** [comment]
Here is the bug in settings regarding drop down lists more explained, I hope you will understand now, because I don't know, how to make it clearer... http://puu.sh/q8jH1.png  
Thank you for your fast responce and resolving that many bugs. shame that APIs are sometimes messy :(

**Comment by Mordonus** [comment]
You cannot select stuff from dropdowns?

This is how it's supposed to look:

![smartgrid](https://cloud.githubusercontent.com/assets/6102530/16998325/36c6c9d0-4eb9-11e6-8376-864bdcc7191c.gif)
![smartgrid](https://cloud.githubusercontent.com/assets/6102530/16998432/ae9492da-4eb9-11e6-9ccb-f9f3be096d49.gif)

**Comment by Kimod** [comment]
It's a minor thing, it isn't a big deal. I'm explaining my problem with this in this gif, the only problem I have with this, is that those lists are a little inconsistent, one si keeping the same width, while the others don't
![settings](https://cloud.githubusercontent.com/assets/6088494/17001057/a52d2e26-4ec5-11e6-9f45-561bc0a1bfd2.gif)

**Comment by Mordonus** [comment]
Ooh I get it now... It's stock behavior of combo box control and I didn't find any options to customize it. Changing it may prove more problematic than its worth ^^

**Comment by Kimod** [comment]
When I right click/long hold on Image in Article in "Articles", I would like an option to save it
:question: How would you like to save it?

Well, on deskop, you can already save the image of the anime cover art, it opens save dialog and you can save it, so on deskop you can open save dialog and save it where you want it
On mobile, you can't even save the anime cover art, so that's something, you forgot to add? Because for example in readit, when you click on "Save image as", then it opens file manager in the app. Also in readit, there is an option to "Save image" which saves the image to special folder in Images folder, so this is what i meant. Also for gifs, in readit, there is an option to "save gif as video", but I don't know why it does it this way, isn't .gif good enough?

**Comment by Mordonus** [comment]
Hmm, it may be impossible. When rendering html (article in this case) special control is used which behaves like web browser and I have **very** limited options of its customisation (I even cannot change the look of scrollbar). You cannot save images in readdit when actual website is loaded unless it detects that link leads to single image and renders it in different way.

**Comment by Kimod** [comment]
I know you didn't ask for more bug reports, but I had a few bugs, that I forgot about in the original or discovered recently.

**Bugs**
	1 When you click on "Select one at random" it's still a little buggy with selecting the items in the list for me (sometimes stops working and I have to click on the list again to fix it), but you probably noticed this when you tried to fix that "bug" what i had with it (it was more like suggestion tho…), if it works fine for you, try reopening the app and doing it again, I can't 100% reproduce this 
	2 When you fresh open the app, the scroll bar isn't at the edge of the window, you can only get it after you click on one show from your list - http://puu.sh/q9LVT.gif
	3 No "&" in Panty & Stocking with Garterbelt in the title on that right viewer - http://puu.sh/q9Msk.png
	4 When you are viewing someone's anime list, it shows my favorites, not theirs (maybe this is intentional, because you said, that pulling favorites is difficult?)
	5 You probably know about this and you don't see a simple solution, but I will list it anyway. At the start, the scrollbar is big, but when you grab it and move down, it starts shrinking, so if you grabbed it low enough, you won't reach the bottom in single drag - http://puu.sh/q9NiJ.gif (this is also a problem, when you just use scroll wheel, it's a little choppy when you haven't loaded all of the shows in that list)
	6 So, this is really something that is pointless and dumb but I will list it here, because it bugs me. When you press Tab and it is selecting hamburger menu, it is inconsistent in selesting things in those flyouts, in Anime List and Manga List, it selectes only one and then you can move by arrows, while in Top Anime it goes through them all, it also tries to selects those lines (for example that line between Calendar and Manga List)(sorry if I'm submiting pointless things :D) gif if you didn't even get what I'm talking about - http://puu.sh/q9Pjq.gif
	7 When you change "My Start" or "My End" on anime and then reopen app, it still updates on the site, but the app has cached the old date, so it will show My Start and My End unchanged (so only visual bug that fixes itself, when the cashed data expires)

**Suggestions**
	1 In profile, if you don't have any favorite manga or people (or other things), wouldn't it be better, if it wouldn't show the whole thing? Because this looks really dumb: http://puu.sh/q9PFN.jpg

So the next two will probably take you a long time, so you can put them on-hold, just the idea that you will consider and maybe add them in the future will make me happy. (Currently I'm really glad you inplemented characters in show details ^^)

So I still use the website for two things that your app can't do, so if you would implement those two things, you would make me really happy, but you would probably have to pull some things from html
	2 So every Saturday on midnight I go to this list (http://myanimelist.net/history/Kimod/anime) to see, what and how much I actualy saw in the last 7 days, you could probably pull that data from website or somehow store what changed and when it changed in app, but it will probably be easier to pull this data from html, you could even make a notificaton like: "See what you watched in the last week"
	3 And the second thing is comparing lists, if you don't know what am I talking about (but you probably do), it's this: http://myanimelist.net/shared.php?u1=Drutol&u2=Kimod&type=anime I think you could come up with something that could work… Alternative and more simple solution would be to show your score when you are viewing their list (in that place, where + and - is, because you can never change their amount of watched episodes)

**Comment by Mordonus** [comment]
Is this big mobile bug still a thing? I believe I addressed it in last update... + it works on my machine ;d

**Comment by Kimod** [comment]
That mobile bug is still present on my device, according to my research, it has to do with scaling of the display, it will load on 150%, so it works on that scaling, but when I have the display in 200% scailing, it won't work...

**Comment by Mordonus** [comment]
了解

**Comment by Mordonus** [comment]
I was planning to send you new build today but I'm too tired to package everything now... Expect them tommorow :)

**Comment by Mordonus** [comment]
Okay, now I'm going to look into history

**Comment by Mordonus** [comment]
In the meantime...

Here are promised builds:

x64:
https://1drv.ms/u/s!Am1xQK-HVf_IkdJe2T917pFMChEhnQ

ARM:
https://1drv.ms/u/s!Am1xQK-HVf_IkdJf3uMnxmR-5yTxBg

**Comment by Kimod** [comment]
Daily dose of new bugs ^^

**"Big mobile bug"**
* So now when I view the list in Detailed Grid mode, it will now list exactly 10 shows instead of 2 shows, so progress? :D And also there is an issue in Grid mode. When I move too fast down, the shows stop loading. Here are both bugs shown - http://puu.sh/qj6bh.gif

**Bugs**
* (Mobile) You didn't make the title selectable http://puu.sh/qhhrq.png
* The scrollbar is correctly placed at the start, but when you select a show and then you click on X, the scrollbar dissapears http://puu.sh/qj6HS.gif
* You can select "Setings - <version>" - http://puu.sh/qj6Nh.png
* There isn't scrollbar in "General" and "About" settings and the Up and Down arrows switch the Color theme instead of scrolling in General settings
* When I try to select show really fast after the last one, or on some rare ocations, the Image and Synopsis won't get updated. In this gif it will happen 2 times, first is nearly at the start and the second is like 1 minute in… - http://puu.sh/qj7Y5.gif
* You could maybe add a little logic to figure out, where to place those boxes, instead of making it 2x2. http://puu.sh/qj8hR.png Only in Favs, I get that you want to have favs and Recent updates separated. While we are talking about this, couldn't you move Recent Updates above Favs? Maybe this is only me, but I will more often check Recent updates than Favorities on friends profile
* When the dialog for setting the Status from Watching to Completed appears, the Status on that right viewer won't update (but when you manualy change the number to match the amount of episodes, it works as intended, so it is bugged only if you use the '+' button) - http://puu.sh/qj8Wc.gif

**Comment by Mordonus** [comment]
About history... what do you think about this organisation? Each section (today, last week etc.) will have grouped anime/manga and in these groups all relevant updates will be displayed.

(Just displayed raw processed data)
![image](https://cloud.githubusercontent.com/assets/6102530/17270387/c29ee262-5661-11e6-98dc-50371b5e82c8.png)

**Comment by Kimod** [comment]
Looks good, maybe one thing that I think would improve this, is adding the episode number next to the date, because you can increment the emount of watched episodes simply by pressing '+' or by entering the emount of watched episodes. So if you, for example watched 2 eps, you would click on '+' 2 times and in this history it would show the date 2 times. Later you could watch like 10 episodes, and maybe you are too lazy to press '+' 10 times, so instead you would enter the emount of watched episodes. That would show as one date in the history. So adding the watched episode number next to the date would solve this...

Otherwise it looks pretty solid ^^

**Comment by Mordonus** [comment]
Let me do the layout and I'll post ss once I'm done :)

**Comment by Mordonus** [comment]
Something like this?

![image](https://cloud.githubusercontent.com/assets/6102530/17271206/319597d8-5676-11e6-9170-ea8bc8dadc78.png)

**Comment by Kimod** [comment]
Looks good, really good ^^

But the scollbar isn't at the edge... :D

**Comment by Mordonus** [comment]
Here you go:

![image](https://cloud.githubusercontent.com/assets/6102530/17271506/fa1ede24-567d-11e6-9a58-c79498b606e5.png)

**Comment by Mordonus** [comment]
Okay, this the build with fixes from your last list + history + profile comments.

I don't know how bugged this is but you seem to have natural talent for finding bugs/issues. And I've little time to check this now thoroughly. I want to upload this version feature wise to the store tommorow ^^

x64
https://1drv.ms/u/s!Am1xQK-HVf_IkdMGLdSF12rZOZAODw

**Comment by Mordonus** [comment]
Uhuh I see crashes ;d

**Comment by Kimod** [comment]
Oh I will post this commend now when you are here and maybe edit it later if I find something later :)

* The only problem with this build is that when I view somebody's history from profile, it crashes (but not always). First it crashed even for my history list, when I viewed from my profile and not through hamburger directly. http://puu.sh/qkBPs.gif
* And also, you fixed the scrollbar Issue from my third list by making it like it was before on my second list, so at the start it isn't on the edge and when I select a show it is at the edge. So I should probably stop worrying about bugs, where scrollbar isn't at the edge, when he should, right?

Love the history though, better than I imagined ^^

If you have time for a few issues that I recently discovered
* In recommendations, the details for the show on the right are on the right too much, so if you have that right viewer opened, it covers your's details - http://puu.sh/qkCbb.png (the recommendation is also worth reading... :D)
* In top anime, when you don't have a show in your list, it doesn't show the show's type, even after I click on it - http://puu.sh/qkCpt.png
* When somebody doesn't have a profile picture, it is blank, so maybe add something like on MAL?

It is harder and harder to fing bugs/issues :D

**Comment by Mordonus** [comment]
How many times did this app crash on most recent build?

**Comment by Kimod** [comment]
Only sometimes when I opened History, elsewhere it didn't crash at all, well I have one bug regarding empty list and clicking on Select one at random, but I believe that it was there before

**Comment by Mordonus** [comment]
So it didn't crash ...
![image](https://cloud.githubusercontent.com/assets/6102530/17276078/03af889e-571e-11e6-966e-38d2e2b98afd.png)
... 29 times?

**Comment by Kimod** [comment]
Well, it is possible... :D I tried to find reliable way to make it crash, but didn't find any

**Comment by Mordonus** [comment]
Okay, I was hoping that crash analytics on windows store were more accurate xd

**Comment by Mordonus** [comment]
I'll upload next build in a few minutes and I'm off to preparing mobile views.

**Comment by Mordonus** [comment]
Okay, play with this and I'm going to start working on mobile...

x64 4.4.11.0
https://1drv.ms/u/s!Am1xQK-HVf_IkdZiIIjbT12-cSQ3gw

**Comment by Kimod** [comment]
ok, it doesn't crash now, which is always good, right?
* Sometimes history doesn't load and also you have a little typo - "epmty" - http://puu.sh/qkMu8.gif
* The back arrow stays there when I go from history to my "Anime List". And when I view my history through my profile, the back arrow is there, but when i view my history through that icon in hamburger menu, the back arrow isn't there - http://puu.sh/qkMCi.gif

Sorry for the delay, I was eating lunch ^^

**Comment by Mordonus** [comment]
First mobile build with new stuff:
(I'd like to treat these like RC's without changelog but we all know how this will turn out...)

ARM v4.4.14.0
https://1drv.ms/u/s!Am1xQK-HVf_IkdZjnvZfo5-3ZfuZ2w

x64 v4.4.13.0
https://1drv.ms/u/s!Am1xQK-HVf_IkdZkScENmvUmNZHslQ

**Comment by Kimod** [comment]
First I tested mobile
Here is how I see the history tab, http://i.imgur.com/527x2gs.png the first two are how I see it, unfortunaly, it is too big, when I see it in my small scalling. The one below is a little concept, that proves, it can fit, if the borders aren't that big
And can confirm that Detailed Grid and Grid mode now work flawlessly ^^

**Comment by Mordonus** [comment]
Fixed this mobile history:
```
03.08.2016 -  v4.4.16.0 & v4.4.15.0

Added profile comments.
Added profile history.
Improved organisation of profile page.
Gave some love to light theme.
Search will be now more accurate and there will be delay after typing in query.
Resolved issues with being unable to load more items on mobile.
Selecting at random will be now smarter.
Fixed visual bug with start/end dates.
Fixed bug where refresh button could create duplicates.
Status bar will now correctly show show titles and other headers.
You can now select and copy text from header in anime details.
Navigation mechanisms have been rewritten.
Profile page will no longer show empty favourites.
Resolved with adaptation issues in recommendations page.
```

**Comment by Kimod** [comment]
Found some bugs and also bugs from earlier lists (I did this because I wanted them to be in one place ^^)

* There is pin my profile, it appears/dissapears 100% if you reproduce the same things in my gif - http://puu.sh/qmYRS.gif (My profile, click on friend's profile, click on my profile (in my friends friends), refresh - pin button there!)
* Scrollbars in my "Anime List" in Compact list, in profile and in history aren't on the edge after fresh open
* Details under Recommendations weren't fixed completly - http://puu.sh/qmZgy.gif (If you had one show selected before you went to the recommendations, the data in details would never go under the each other)
* I can't disable "Enable swipe to increment/decrement on gird items" - http://puu.sh/qmZMO.png
* That anime viewer on right shouldn't have back button when you first open show details (when you had one show shown before) (Steps: Click on one show, close the right viewer and then click on another show, now that another show has back button that leads to the first show) Experiment with this some more, like clicking on more shows before closing the viewer and then the back button. Gif that shows something I just described - http://puu.sh/qn0SA.gif
* When I click in my profile on History (not in hamburger menu, in my profile), it also creates that back arrow (but if I click on Anime List/Manga list in my profile, it doesn't create one, so inconsistency? Sorry if lately I'm just pointing out pointless stuff ^^)
* Also when you were making white theme prettier, you missed this - http://i.imgur.com/E1hMNf0.png (You should probably make the star white, so it matches the white text on gray background ^^)
* Settings in windows 10 and other UWP applications show category when you click on them when you are in that category, your app still shows Settings - (version) - http://puu.sh/qkEeu.png
 * 💮 I'm to lazy to make switch for all those pages right now... nanana
* Bigger thing - Could you add to the settings select a timezone? You could use it in history - http://puu.sh/qkERZ.jpg (on MAL I have correct timezone) and in airing days in list, because when I move it by 1 in settings, it is too much (for example I watch Re:zero on Sunday (Broadcast: Mondays at 01:05 (JST)) and Shokugeki no Souma: Ni no Sara on Saturday (Broadcast: Saturdays at 22:00 (JST)) (they are 1day, 3hours and 5 minutes appart. In Japan they are between 2 days while for example in my timezone they are between 1 day). This is maybe bigger thing but it would add something that would make your app better than the website
 * Let's talk about sliders maybe...? I'll have to think about it because it's not trivial matter.

Some mobile only bugs
* When you are in subcategory in settings, when you press back button, it closes the settings completly. When it should go to the main menu of settings
* I believe that my Start page should also be the last page before the app closes when I click on back button, but if I'm in "Seasonal Anime", "Top anime" or "Messages",  and click on back button, it closes the app
* This http://puu.sh/q7sgO.jpg happens in other places: http://puu.sh/qn0B3.png
*  Settings inconsistency between desktop and mobile - http://puu.sh/qn0ca.png (Desktop - Start page -> Color theme, Mobile - Color theme -> Start page
* Profile without picture doesn't have "?" - http://puu.sh/qn0iA.jpg
* History tab almost there… - http://puu.sh/qn1Eo.jpg

**Comment by Mordonus** [comment]
I'll resolve these over the weekend as I'm working on demo version of my internship app right now. Additionally I'm planning to either add character details or forums in "beta" version. And those dreaded ads will appear next week...

**Comment by Mordonus** [comment]
Once more... ***screw* ads**. 

Fixed some things, some of them I don't even want to touch and some are left for tommorow :) Because of new AU emojis we are switching white empty checkmark to :white_flower: because why not :D

**Comment by Kimod** [comment]
It's that time again! More bugs, Yeeey ^^

* Wasn't fixed for me - When manga has 0 chapters/volumes read, it shows "? Episodes" instead of "? Read" -http://puu.sh/qq3Ws.png
* When profile doesn't have profile picture, it doesn't show profile details (names to test on: THAT_IS_NOT_OKAY, Ahrichards5241) - http://puu.sh/quizm.png
* When you click on "Profile" in hamburger menu, it should delete that back button on top, because if not - http://puu.sh/quiGT.gif
* Articles and News doesn't have a scrollbar at all
* The airing day in "Anime List" disappears if you go to "Top Anime - Airing" - http://puu.sh/qz0UP.gif
* Capital letters in user names aren't capital, only if you are looking at someone's list/profile - http://puu.sh/qz1wg.gif (Desktop only)
* Arrow to top disappears after you click on a show - http://puu.sh/qz2Li.gif
* "Last watched" wasn't implemented, or why is it grayed out? - http://puu.sh/qz3vS.png (in general settings)
* That pivot which you implemented for "Watching" "Completed" etc. doesn't have those left/right switching arrows and also the most right item is higher that the others - http://puu.sh/qz42R.gif
* If you have subcategory in settings opened and then click on anime/manga, it does have that back arrow - http://puu.sh/qz4iV.gif
* In profile, clicking on a show brings both back arrows - http://puu.sh/qz4zg.gif
* When you select watching on a show, it does select My start, but if I click on "+" and it asks me to switch it to watching, it doesn't select My start - http://puu.sh/qAjtm.gif
* In Synopsis, on some shows instead of dashes are semicolons - http://puu.sh/qz55I.jpg , http://puu.sh/qz5mg.png or some other special characters - http://puu.sh/qz5II.jpg
* When reviews are loading and click on refresh button, it loads the reviews multiple times - http://puu.sh/qz6sO.gif
* That place in Compact list where +/- are, are taking unnecessary space, when you are viewing somebody else's list - http://puu.sh/qz959.png
* Tags on Detailed grid mode are behind the text http://puu.sh/qzK5i.jpg (you could probably move the tag below the star)
* The dialog when you right click on selected text is white - http://puu.sh/qA7h3.png
* If I click on "anime list" in someone's profile and then go back with that back button, I'm back at their profile, whereas if I click on "manga list" and then go back, I'm back in my profile - http://puu.sh/qzKiX.gif
* When you extended one review and then clicked on different show and go to reviews, it has one review extended - http://puu.sh/qAiJG.gif

**Not really bugs**
* When you close settings, if you had something selected in hamburger, select it back - http://puu.sh/qAjKu.gif
* When that right viewer is at 0 width and click on the same show that was previously selected, it doesn't extend it - http://puu.sh/qz9Fv.gif (maybe when it is at 0 width, just close it? more touch friendly closing)
* When you click on History or Messaging, it should select that icon in hamburger menu - http://puu.sh/quiDO.png
* A way to modify tags (deleting and re-writing them is too much work... :D)
* Some shows have their airing day still after they are done airing - http://puu.sh/quioF.png , you could probably somehow check for end day, then for what day today is and if today > end day, don't show airing day, or maybe just don't show airing day in Completed (I don't know why I even bothered with writing the solution, because you can figure this out on your own and also you know what is/isn't possible more than I do, so writing solution is pointless… ^^) 
* If I were you, I would focus next (of course after you are done with character details ^^) on "Details" page in details page, because now it's pretty much useless (you could put there the episode length, producer, show popularity and ranked number, which I requested in my first comment ^^. And also you said that you will redo details page in details page earlier :p) and also it's broken (now I don't mean "Data unavailable") for some shows - http://puu.sh/qz7GM.png so I guess just scrap the details page you have now?

**Mobile only**
* I can't load somebody else's history, it shows empty every time
* If you click in your "Profile" page on "Anime list" and then click back, it goes back to "Anime list", so nothing happens
* Clicking back in forums main page closes the app

Also if you were wondering, that thing with different heights on favorite anime and characters in profile when I was switching monitors wasn't fixed, you can skip this, just letting you know ^^

**Comment by Mordonus** [comment]
How is/was railgun/index?

**Comment by Kimod** [comment]
Ohh, I really enjoyed Index and I think that I'm enjoying Railgun even more, so many best grillz :3 I really like how it's lighthearted and the comedy was done just right 👌👌

**Comment by Mordonus** [comment]
Yup, many best girls ^^. I deem first railgun the weakest of four but still very good nontheless. I'm even going through light novels now and next contestant is making her appearance. Let's hope for index3 :D

**Comment by Mordonus** [comment]
I also feel like this list is becoming too long, how about creating new issue next time?

**Comment by Kimod** [comment]
You are probably right, I didn't want to spam your issues with my lists ^^ So next time, instead of commenting with the list, I will create new issue.. And a little question: Should I copy previous bugs to new issue, if you didn't fix them and didn't even coment on them. for example that issue with scrollbar not being on the edge?

**Comment by Mordonus** [comment]
Yeah, copy them just in case...

![image](https://cloud.githubusercontent.com/assets/6102530/17708351/e38e5116-63e4-11e6-85f2-bd9cf5b7a75a.png)

This update will take a while because of the amount of various stuff I'm changing :)

**Comment by Mordonus** [comment]
New build available in #29. Open new issue if you happen to find something (what am I even talking about ^^ )
